### ISSUE TITLE
[Bug]: subtitle parts that have same key dont clear

### ISSUE THREAD

**Issue by RihanArfan** [movie-web/movie-web#295]
### What happened?

After malformed/multiple subtitles show at once, they're not cleared.

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### Steps to reproduce?

1. Series
2. Mob Psycho 100 (https://movie-web.app/media/JW-show-39013/98611/1636826)
3. Season 2
4. Set English subtitles
5. See 1m:35s - 1m42s for a large number of subtitles (expected) which don't clear (bug)

### Other relevant information

Viewing the media directly on FlixHQ, the subtitles disappear when the scene changes

**Comment by zisra** [comment]
<img width="1458" alt="image" src="https://github.com/movie-web/movie-web/assets/100528712/2af3ecf8-a4a2-4085-a933-bb0338b3645b">
🤔

**Comment by frost768** [comment]
```
34
0:01:40,40 --> 0:01:43,61
Executing the Exorcism!!

35
0:01:40,40 --> 0:01:43,61
Refer us to a friend now and get an extra 5% reduction!! 
Also get an add-on exorcism for 20% off!!

36
0:01:40,40 --> 0:01:43,61
We will do our best to reduce the spirit possessing you
Option A — Trial Course\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hA Mere 2,000 yen (Tax Excluded)
\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hOutside Offer Period: 3,980 yen
Option B — Serious Course \h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hA Mere 5,000 yen (Tax Excluded)
\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hOutside Offer Period: 8,980 yen
Option C — All-Out Course \h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hA Mere 12,000 yen (Tax Excluded)
\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\h\hOutside of Offer Period: 19,880 yen

37
0:01:42,69 --> 0:01:43,61
Right...
```

All of them has the same end time. I have never seen `\h` used as a special character

**Comment by zisra** [comment]
Is that an advertisement?

**Comment by frost768** [comment]
The paper on the table in the video is.

**Comment by mrjvs** [comment]
This has been fixed in the latest version: #530
